### PR TITLE
revision of coordinateSpace according to #40

### PR DESCRIPTION
**Pull request by lzehl**
Please review and double check.

### CODE REVIEW

File: v0.1/core/coordinateSpace.schema.json [hunk 1]
```diff
@@ -19,38 +28,34 @@
       "type": "string",
       "description": "User-defined identifier, often a short name, of this coordinate space."
     },
-    "fullName": {
-      "type": "string",
-      "description": "A descriptive name of this coordinate space."
+    "axes": {
+      "description": "Link to axes that are presented in the coordinate space."
```
Comment (UlrikeS91) [inline v0.1/core/coordinateSpace.schema.json:32]: comma missing

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]
It's just that one silly little comma that is missing ;)

**Comment by UlrikeS91** [comment]
Also, Wiki needs to be updated. It's not reflecting these changes 100%.

**Comment by lzehl** [comment]
I'll update the Wiki after the PR is merged. Thanks for the reminder

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**